docs/ru/engines/table-engines/integrations/rabbitmq.md

```diff
@@ -1,8 +1,3 @@
----
```

Comment:
Removing of TOC priority looks strange.

Reply:
I noticed that It is not added for any of the RU doc for table engines